src/special/webpack.js

```diff
@@ -1,12 +1,16 @@
+/* eslint-disable no-console */
+
 import path from 'path';
 import lodash from 'lodash';
 
 const webpackConfigRegex = /webpack(\..+)?\.config\.(babel\.)?js/;
 const loaderTemplates = ['*-webpack-loader', '*-web-loader', '*-loader', '*'];
 
 function extractLoaders(item) {
-  if (item.loader) {
+  if (item.loader && typeof item.loader === 'string') {
```

Comment:
I am thinking about if we can check `item` is a string, then directly return it. If that is possible, we don't need to have `.map(loader => ({ lodaer }))`